### PR TITLE
Maintain/documentation fixes

### LINKED ISSUE #718: [MAINTAIN] Minor fixes and updates to documentation page
### Please describe what needs to be maintained?
Make minor fixes and updates to the documentation page.

### PR DESCRIPTION
**Pull request by RuanJohn**
## What?
Minor typo fixes and changes to the current Mava documentation. 
## Why?
Make the documentation more readable and fix some small typos. 
## Extra 
There are some changes to the Mava readme made in the PR. It will be better to first approve and merge #716 before merging this PR. Current version of the docs from this PR are available at [this link](https://id-mava.readthedocs.io/en/feature-mava-docs-add-on/)
close #718

### CODE REVIEW

**Review by AsadJeewa — COMMENTED** [review]
Comment/ request on consistency. Thanks for this @RuanJohn

File: examples/jax/quickstart.ipynb [hunk 1]
```diff
@@ -269,15 +269,15 @@
         "\n",
         "`optimizer` - the optimiser used by the trainer to updated the agent weights.\n",
         "\n",
-        "`run_evaulator` - a flag indicating weather a seperate environment process should be run that tracks the system's performance using Tensorboard and possibly gameplay recordings.\n",
+        "`run_evaluator` - a flag indicating weather a separate environment process should be run that tracks the system's performance using Tensorboard and possibly gameplay recordings.\n",
```
Comment (AsadJeewa) [inline examples/jax/quickstart.ipynb:272]: "whether" typo

**Comment by AsadJeewa** [comment]
Additional request: remove the paper link to DIAL in mava/systems/README.md as we have removed it from MAVA. This README is displayed here:[https://id-mava.readthedocs.io/en/latest/systems/general/]

**Comment by AsadJeewa** [comment]
> Additional request: remove the paper link to DIAL in mava/systems/README.md as we have removed it from MAVA. This README is displayed here: https://id-mava.readthedocs.io/en/latest/systems/general/

Done in commit above

File: examples/jax/quickstart.ipynb (hunk 1 above)
Comment (AsadJeewa) [reply examples/jax/quickstart.ipynb:272]: Done

**Review by AsadJeewa — DISMISSED** [review]
Thanks @RuanJohn

File: mava/systems/jax/ippo/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Independent Proximal Policy Optimization (IPPO)
 
-This system is an extension of the Proximal Policy Optimization (PPO) ([Schulman et al., 2017]) algorithm for the multi-agent setting. The system is currently trained in a decentralised manner and is hence termed IPPO. Centralised training, termed MAPPO, will be realeased shortly. For recent experimental results concerning Multi-agent PPO, see the following paper by [Yu et al. (2021)].
+This system is an extension of the Proximal Policy Optimization (PPO) ([Schulman et al., 2017]) algorithm for the multi-agent setting. The system is currently trained in a decentralized manner and is hence termed IPPO. Centralized training, termed MAPPO, will be released shortly. For recent experimental results concerning Multi-agent PPO, see the following paper by [Yu et al. (2021)].
```
[the remark this answers is not in the document]
Comment (RuanJohn) [reply mava/systems/jax/ippo/README.md]: Totally agree, I was using an American spell checker. So I will update everything accordingly.

**Review by DriesSmit — APPROVED** [review]
The changes look great! Thanks @RuanJohn :rocket:

**Review by arnupretorius — COMMENTED** [review]
Thanks @RuanJohn! 🙌  Just see my minor comments.

File: docs/getting_started/overview.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Getting started with Mava
+
+> 🚧 **Note:** This only applies to the callback redesign of Mava.
+
+Going from research papers to code and being able to quickly test experimental hypotheses are some of the main challenges faced by reinforcement learning researchers. Mava aims to remove this constraint and to be the go-to framework for fast iteration and experimentation. Mava also effectively utilizes computational resources by leveraging [Launchpad](https://github.com/deepmind/launchpad) for distributed computation.
+
+### Design Paradigm
+
+Mava makes use of the a callback-based approach to enable modularity and flexibility in system designs. This enables the development of new systems with minimal code reuse and overhead leading to fast experimentation and results. The system's architecture is based on components that are overwritten in callbacks, and these callbacks are passed as arguments to the system classes, allowing users to configure systems differently depending on an implemented algorithm's properties. More information on callbacks may be found [here](../components/components.md).
```
[the remark this answers is not in the document]
Comment (RuanJohn) [reply docs/getting_started/overview.md]: Good catch. Done :)

File: mava/systems/jax/ippo/README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Independent Proximal Policy Optimization (IPPO)
 
-This system is an extension of the Proximal Policy Optimization (PPO) ([Schulman et al., 2017]) algorithm for the multi-agent setting. The system is currently trained in a decentralised manner and is hence termed IPPO. Centralised training, termed MAPPO, will be realeased shortly. For recent experimental results concerning Multi-agent PPO, see the following paper by [Yu et al. (2021)].
+This system is an extension of the Proximal Policy Optimization (PPO) ([Schulman et al., 2017]) algorithm for the multi-agent setting. The system is currently trained in a decentralised manner and is hence termed IPPO. Centralised training, termed MAPPO, will be released shortly. For recent experimental results concerning Multi-agent PPO, see the following paper by [Yu et al. (2021)].
```
[the remark this answers is not in the document]
Comment (RuanJohn) [reply mava/systems/jax/ippo/README.md]: Done :)

**Review by DriesSmit — APPROVED** [review]
Thanks @RuanJohn :fire:

**Review by AsadJeewa — APPROVED** [review]
Thanks @RuanJohn 
Not sure if you anna wait for @arnupretorius to approve , based on his comments, though you did address them already

**Pull request merged**